Added a section to describe Terraform State import (#332)
spacelift-io/user-documentation: docs/vendors/terraform/state-management.md

![](<../../assets/screenshots/Screen Shot 2022-02-15 at 1.31.29 PM.png>)

## Importing existing state file into your Terraform Stacks

> If you are managing your infrastructure resources with Terraform prior to start using Spacelift, you would probably have an existing Terraform State file that you manage yourself. You have the option to import your existing Terraform State files into Spacelift and let Spacelift manage your states for you.

Reviewer: What about the following?

```suggestion
When creating a stack, you can optionally import an existing Terraform state file so that Spacelift can manage it going forward.
```

> You can import your existing Terraform State files in Stack creation process.

Reviewer: This can be removed if the suggestion above is accepted.

```suggestion
```

> ![](../../assets/screenshots/Stack_settings_importing_state_file.png)

Reviewer: The screenshot uses chalk orange lines, while the other screenshots use solid red lines. Please make this screenshot consistent with the rest of the documentation.

More generally, we want to approach editing this documentation like changing legacy source code: try to make your changes blend in nicely.

Author: Done
